README.md

```diff
@@ -94,12 +137,12 @@ In the following table you can find what are supported Kubernetes and Terraform
 versions for each KubeOne version. KubeOne versions that are crossed out are not
 supported. It's highly recommended to use the latest version whenever possible.
 
-| KubeOne version | 1.14 | 1.13 | Terraform | Supported providers                                                |
-|-----------------|------|------|-----------|--------------------------------------------------------------------|
-| v0.9.0+         | +    | +    | v0.12+    | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack, vSphere, Azure |
-| v0.8.0+         | +    | +    | v0.11     | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack, vSphere        |
-| v0.6.0+         | +    | +    | v0.11     | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack                 |
-| v0.5.0          | +    | +    | v0.11     | AWS, DigitalOcean, GCE, Hetzner, OpenStack                         |
+| KubeOne version | 1.15 | 1.14 | 1.13 | Terraform | Supported providers                                                |
+|-----------------|------|------|------|-----------|--------------------------------------------------------------------|
+| v0.9.0+         | +    | +    | +    | v0.12+    | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack, vSphere, Azure |
+| v0.8.0+         | -    | +    | +    | v0.11     | AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack, vSphere        |
```

Comment:
I think it's possible to install 1.15 using v0.8 version... and probably even previous version as well.

Reply:
It is but I'd not recommend it considering we had some needed changes that are not backported. What comes to my mind is the default admission controllers.